Reinstate Google form

fureigh opened the pull request:
Follow-up on #335, which has been reverted.

@xtine, the remaining task is to troubleshoot why form submissions aren't registering in the appropriate Google doc. @hestherchang is done for the day but I've asked her to grant you access to the [spreadsheet of responses](https://docs.google.com/spreadsheets/d/1T7oi055_4hHWoUHyxt1l5m6SqKqi-mAF0UbfLnjr5Rw/edit#gid=1455047377) once she's back.

fureigh committed:
Restore old Google Form application

From 9beed21. Next step is for @hestherchang to update it. :tada:

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Restore application confirmation page

From 9beed21.

https://github.com/18F/joining-18f/issues/334

hestherchang committed:
updated fields to reflect new application form

fureigh committed:
Update destination of form submissions

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Add stub page for product lead position

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Rename 'product lead' to 'product manager'

Per @hestherchang's feedback.

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Update product manager position's permalink

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Adjust fields' copy, generally to match the Google form

Also includes making 'Preferred name' an optional, not required,
field.

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Change some fields from checkboxes to radio buttons

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Add indicators for remaining required fields

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Copy edit: US --> U.S.

https://github.com/18F/joining-18f/issues/334

hestherchang committed:
Update product-manager.md

final job posting! would appreciate another look

hestherchang committed:
Update product-manager.md

updated # of vacancies from 1 to 3

fureigh committed:
Copy edit (position and GS level)

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Lowercase a bunch of capitalized words

...per 18F's content guide.

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Remove remaining `<br>`s from Markdown file

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Copy edits to 'number of vacancies'

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Add application link to end of position description

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Add Product Manager position to open roles page

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Update name fields per conversation

The problem was that people have been putting in just one word
(ex: 'Dave') despite using more than one word as their usual name,
and that's been making things confusing for the Talent Team.
Hopefully this will be clearer.

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Revert application confirmation message

...from the longer, friendlier one to a simpler one, because the
application confirmation screen should include the relevant text.

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Add 18F-and-GSA explainer line

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Cleanup: remove Lever-specific assets

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Update apply link

This may be an artifact of testing locally.

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Add application open and close dates

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Capitulate to apply.html

https://github.com/18F/joining-18f/issues/334

xtine committed:
change header border to match new logo color

xtine committed:
update form actions

xtine commented:
Got the form to successfully submit to the google spreadsheet. I updated some of the hidden inputs to the source of the original live google form and also added required attributes to Eligibility, Phone, City, State that were optional before.

fureigh committed:
Bugfix: make sure DC saves as an option

Previously, 'Washington, D.C.' was silently omitted from results.

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Improve Product Manager formatting

* Add trailing whitespace to force linebreaks
* Add additional linebreaks to make bulleted lists render
* Convert URLs to links

https://github.com/18F/joining-18f/issues/334

fureigh committed:
Add 'aria-required' values to location field options

https://github.com/18F/joining-18f/issues/334

fureigh commented:
Thank you so much, @xtine! I made a few more changes. Merging now per earlier conversation with @hestherchang.
